torch/_inductor/mkldnn.py

```diff
@@ -20,6 +20,14 @@ def is_group_depthwise_conv_transpose(m):
     )
 
 
+# Port from: aten/src/ATen/native/Convolution.cpp:is_output_padding_big
+def is_output_padding_big(m):
+    is_big = False
+    for i in range(len(m.output_padding)):
+        is_big |= m.output_padding[i] >= m.stride[i]
+    return is_big
```

Comment:
a more pythonic impl and you don't need an extra function:
`any(padding >= stride for padding, stride in zip(m.output_padding, m.stride))`